master-docker-nonstandard/master.cfg

```diff
@@ -222,6 +222,9 @@ addWorker('hz-bbw', 1, '-msan-clang-debian-11', "quay.io/mariadb-foundation/bb-w
 addWorker('hz-bbw', 4, '-msan-clang-debian-11', "quay.io/mariadb-foundation/bb-worker:debian11-msan", jobs=20, save_packages=False)
 addWorker('hz-bbw', 5, '-msan-clang-debian-11', "quay.io/mariadb-foundation/bb-worker:debian11-msan", jobs=30, save_packages=False)
 
+addWorker('hz-bbw', 2, '-debian-12', "quay.io/mariadb-foundation/bb-worker:debian12", jobs=20, save_packages=False)
+addWorker('hz-bbw', 5, '-debian-12', "quay.io/mariadb-foundation/bb-worker:debian12", jobs=20, save_packages=False)
```

Comment:
shouldn't these have a `-asan-ubsan` suffix on the worker image name?

On worker name might be good for searchability too.

Reply:
from a manual run that seemed to work, there are no additional packages needed, so we can use the base debian12 image for the asan and ubsan